PathPlanning/DStar/dstar.py

```diff
@@ -0,0 +1,217 @@
+import math
```

Comment:
Could you add header comment which includes short description, author name, and reference paper?